docs/sources/get-started/deployment-modes.md

```diff
@@ -8,101 +8,81 @@ aliases:
 ---
 # Loki deployment modes
 
-As an application, 
-Loki is built out of many component microservices,
-and is designed to run as a horizontally-scalable distributed system.
-Loki's unique design compiles the code of the entire distributed system into
-a single binary or Docker image.
-The behavior of that single binary is controlled with
-the `-target` command-line flag and defines one of three modes of operation.
+Loki has many component microservices, and is designed to run as a horizontally-scalable distributed system. The Loki binary and Docker image include all of the Loki component microservices, letting you customize how you deploy Loki.
 
-Configuration for each deployed instance of the binary
-further specifies which components it runs.
+You can configure the behavior of the single binary with the `-target` command-line flag to specify which microservices will run on startup. You can further configure each of the components in the `loki.yaml` file.
 
-Loki is designed to easily redeploy a cluster under a different mode
-as your needs change,
-with no configuration changes or minimal configuration changes.
+As Loki stores indexes and chunks separately, you can easily redeploy a cluster under a different mode as your needs change, with minimal or no configuration changes.
 
-## Monolithic mode
-
-The simplest mode of operation sets `-target=all`.
-This is the default target,
-and it does not need to be specified.
-This is monolithic mode;
-it runs all of Loki’s microservice components inside a single process
-as a single binary or Docker image.
```

Comment:
I'm reluctant to remove this, most of the folks using Loki in the wild run the single binary.

I think we may just want to make clear this is for low volume hobby setups or low volume installations over a fixed number of sources and small number of users.